libear/CMakeLists.txt

```diff
@@ -31,7 +31,9 @@ if(CMAKE_THREAD_LIBS_INIT)
 endif()
 
 set(CMAKE_MACOSX_RPATH 1)
```

Comment:
Could you include this line into the guarded region? I think these two together fix OSX builds, so would keep them together.